tests/integration/ie-tests.sh

```diff
@@ -14,57 +14,69 @@ test_example_job() {
 
   echo $K8S_MASTER_URL
 
+  PREVIOUS_JOB=$(kubectl --kubeconfig=${KUBE_CONFIG} get pods | grep driver | tail -n 1 | cut -d' ' -f1)
```

Comment:
If possible it'd be good to swap this kind of subshell-foo for lightkube, not immediately a priority, but something for the future.